src/baseTrie.js

```diff
@@ -28,7 +28,7 @@ module.exports = class Trie {
     this.EMPTY_TRIE_ROOT = ethUtil.KECCAK256_RLP
     this.sem = semaphore(1)
 
-    this.db = db || new DB()
+    this.db = db ? new DB(db) : new DB()
```

Comment:
Associated JSDocs need to be updated as well.

Reply:
Good catch. Also had forgotten to re-generate the docs.